develop/dev-guide-sample-application-python-mysqlclient.md

```diff
@@ -1,292 +1,237 @@
+Connect to your TiDB cluster depending on the TiDB deployment option you've selected.
 
-def get_connection(autocommit: bool = True) -> MySQLdb.Connection:
-    return MySQLdb.connect(
-        host="127.0.0.1",
-        port=4000,
-        user="root",
-        password="",
-        database="test",
-        autocommit=autocommit
-    )
+<SimpleTab>
+<div label="TiDB Serverless">
 
+1. In the TiDB Cloud, navigate to the [Clusters](https://tidbcloud.com/console/clusters) page, select your TiDB Serverless cluster. Go to the **Overview** page, and click the **Connect** button in the upper right corner.
 
-def create_player(cursor: Cursor, player: tuple) -> None:
-    cursor.execute("INSERT INTO player (id, coins, goods) VALUES (%s, %s, %s)", player)
+2. Ensure the configurations in the confirmation window match your operating environment.
 
+    - **Endpoint Type** is set to **Public**
+    - **Connect With** is set to **General**
+    - Operating System matches your environment.
 
-def get_player(cursor: Cursor, player_id: str) -> tuple:
-    cursor.execute("SELECT id, coins, goods FROM player WHERE id = %s", (player_id,))
-    return cursor.fetchone()
+    <Tip>If you are running in Windows Subsystem for Linux (WSL), switch to the corresponding Linux distribution.</Tip>
 
+3. Click **Create password** to create a random password.
 
-def get_players_with_limit(cursor: Cursor, limit: int) -> List[tuple]:
-    cursor.execute("SELECT id, coins, goods FROM player LIMIT %s", (limit,))
-    return cursor.fetchall()
+    <Tip>If you have created a password before, you can either use the original password or click **Reset password** to generate a new one.</Tip>
 
+4. Run the following command to copy `.env.example` and rename it to `.env`:
 
-def random_player(amount: int) -> List[tuple]:
-    players = []
-    for _ in range(amount):
-        players.append((uuid.uuid4(), 10000, 10000))
+    ```bash
+    cp .env.example .env
+    ```
 
-    return players
+5. Copy and paste the corresponding connection string into the `.env` file. Example result is as follows:
 
+    ```dotenv
+    TIDB_HOST='{gateway-region}.aws.tidbcloud.com'
+    TIDB_PORT='4000'
+    TIDB_USER='{prefix}.root'
+    TIDB_PASSWORD='{password}'
+    TIDB_DB_NAME='test'
+    CA_PATH=''
+    ```
 
-def bulk_create_player(cursor: Cursor, players: List[tuple]) -> None:
-    cursor.executemany("INSERT INTO player (id, coins, goods) VALUES (%s, %s, %s)", players)
+    Be sure to replace the placeholders `{}` with the values obtained from the connection dialog.
 
+    TiDB Serverless requires a secure connection. Since the `ssl_mode` of mysqlclient defaults to `PREFERRED`, you don't need to manually specify `CA_PATH`. Just leave it empty. But if you have a special reason to specify `CA_PATH` manually, you can refer to the [TLS Connections to TiDB Serverless](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) to get the certificate paths for different operating systems.
```

Comment:
```suggestion
    <CustomContent platform="tidb">

    TiDB Serverless requires a secure connection. Since the `ssl_mode` of mysqlclient defaults to `PREFERRED`, you don't need to manually specify `CA_PATH`. Just leave it empty. But if you have a special reason to specify `CA_PATH` manually, you can refer to the [TLS Connections to TiDB Serverless](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-clusters) to get the certificate paths for different operating systems.

    </CustomContent>
    <CustomContent platform="tidb-cloud">
    
    TiDB Serverless requires a secure connection. Since the `ssl_mode` of mysqlclient defaults to `PREFERRED`, you don't need to manually specify `CA_PATH`. Just leave it empty. But if you have a special reason to specify `CA_PATH` manually, you can refer to the [TLS Connections to TiDB Serverless](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) to get the certificate paths for different operating systems.

    </CustomContent>
```